db/migrate/20200828125910_change_application_start_and_end_date_for_recruitment_cycles.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeApplicationStartAndEndDateForRecruitmentCycles < ActiveRecord::Migration[6.0]
+  def up
+    recruitment_cycle = RecruitmentCycle.find_by(year: "2021")
+    return if recruitment_cycle.nil?
+
+    recruitment_cycle.application_start_date = "13/10/2020"
```

Comment:
I'd be slightly uncomfortable running this or any date in this format. It does work locally but your system probably knows you're in the London timezone and may assume British dates. The server will most likely be in UTC and might not parse this string as you intended.

I'd probably opt for a date object so there is no uncertainty:
```suggestion
    recruitment_cycle.application_start_date = Date.new(2020, 10, 13)
```

Reply:
Thanks Phil, I've updated